S2VX.Game/Editor/ToolState/NoteToolState.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace S2VX.Game.Editor.ToolState {
     public class NoteToolState : S2VXToolState {
+        private Notes PreviewContainer { get; } = new Notes();
```

Comment:
What's this for?

Reply:
It's so the preview note gets updated. The preview note used to work by itself but after I moved all of the update logic into S2VXNotes, a note can't exist without its container since it doesn't have its own update logic